packages/windows/src/RPA/Windows/keywords/context.py

```diff
@@ -33,5 +46,65 @@ def logger(self):
         return self.ctx.logger
 
     @property
-    def window(self):
-        return self.ctx.window
+    def current_timeout(self) -> float:
+        # This value can change based on `auto.SetGlobalSearchTimeout(...)` calls.
+        return auto.uiautomation.TIME_OUT_SECOND
+
+    def _window_or_none(self, window) -> Optional["WindowsElement"]:  # noqa: F821
+        if window and window.item:
+            # FIXME(cmiN): Investigate why `.Exists()` is `False` with Desktop element.
+            #  (`auto.GetRootControl().Exists()` -> `False`)
```

Comment:
I guess the root control always return false as it doesn't have visible window. You can get windows from that by calling:  auto.GetRootControl().GetChildren(). Which then return list of auto.Control objects that has the Name and handle objects.

Another way of getting windows directly from the windows API would be:
win32gui.FindWindow(None, "Calculator")
Though I guess we don't want to have too many libraries to get the windows elements. Just makes things confusing.

It would also be nice to have the type declaration for the parameters. For example here I guess this is: window: auto.Control.

Reply:
Thx!
- Added "WindowsElement" as `window` typing. (since that's what we're receiving and returning)
- Given the access of Windows, I'm only interested into one thing: is the received control (usually app window) still existing on the screen or not? (and testing it with either of the bounding rectangle or children attribute produces the same COMError I was looking for)

```
>>> ca.BoundingRectangle
Rect(77,64,743,1142)[666x1078]
>>> ca.BoundingRectangle
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "Z:\Repos\rpaframework\packages\windows\.venv\lib\site-packages\uiautomation\uiautomation.py", line 5906, in BoundingRectangle
    rect = self.Element.CurrentBoundingRectangle
_ctypes.COMError: (-2147220991, 'An event was unable to invoke any of the subscribers', (None, None, None, 0, None))
>>> ca.GetChildren()
Traceback (most recent call last):
  File "<console>", line 1, in <module>
  File "Z:\Repos\rpaframework\packages\windows\.venv\lib\site-packages\uiautomation\uiautomation.py", line 6341, in GetChildren
    child = self.GetFirstChildControl()
  File "Z:\Repos\rpaframework\packages\windows\.venv\lib\site-packages\uiautomation\uiautomation.py", line 6287, in GetFirstChildControl
    ele = _AutomationClient.instance().ViewWalker.GetFirstChildElement(self.Element)
_ctypes.COMError: (-2147220991, 'An event was unable to invoke any of the subscribers', (None, None, None, 0, None))
```

Now since Desktop makes sense of not having an active Window, I'll just remove the `.Exists()` block to make the test much more simpler. (by testing against the roots of the problem)